Sources/NIO/Embedded.swift

```diff
@@ -391,15 +394,22 @@ public final class EmbeddedChannel: Channel {
 
     /// Synchronously closes the `EmbeddedChannel`.
     ///
-    /// This method will throw if the `Channel` hit any unconsumed errors or if the `close` fails. Errors in the
-    /// `EmbeddedChannel` can be consumed using `throwIfErrorCaught`.
+    /// Errors in the `EmbeddedChannel` can be consumed using `throwIfErrorCaught`.
     ///
+    /// - parameters:
+    ///     - acceptAlreadyClosed: Whether `finish` should throw if the `EmbeddedChannel` has been previously `close`d.
     /// - returns: The `LeftOverState` of the `EmbeddedChannel`. If all the inbound and outbound events have been
     ///            consumed (using `readInbound` / `readOutbound`) and there are no pending outbound events (unflushed
     ///            writes) this will be `.clean`. If there are any unconsumed inbound, outbound, or pending outbound
     ///            events, the `EmbeddedChannel` will returns those as `.leftOvers(inbound:outbound:pendingOutbound:)`.
-    public func finish() throws -> LeftOverState {
-        try close().wait()
+    public func finish(acceptAlreadyClosed: Bool) throws -> LeftOverState {
```

Comment:
This flag doesn’t actually do anything...

Reply:
I think I fixed this already